src/main/java/com/openshift/restclient/ResourceKind.java

```diff
@@ -98,6 +99,7 @@
 		list.add(RESOURCE_QUOTA);
 		list.add(SERVICE);
 		list.add(SECRET);
+		list.add(SERVICE_ACCOUNT);
```

Comment:
You are correct, this should be a set.  Please feel free to change as needed.  The only place this 'list' is used is to validate the registered types have the proper constructor used by reflection in the ResourceFactory.

----- Original Message -----
> From: "André Dietisheim" <notifications@github.com>
> To: "openshift/openshift-restclient-java" <openshift-restclient-java@noreply.github.com>
> Cc: "Jeff Cantrill" <jcantril@redhat.com>
> Sent: Wednesday, June 24, 2015 5:04:13 AM
> Subject: Re: [openshift-restclient-java] Add support for K8 serviceAccount (#37)
> 
> > @@ -98,6 +99,7 @@
> >  		list.add(RESOURCE_QUOTA);
> >  		list.add(SERVICE);
> >  		list.add(SECRET);
> > +		list.add(SERVICE_ACCOUNT);
> 
> @jcantrill to me this list very much like it should be a set
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/openshift-restclient-java/pull/37/files#r33129535